plugin/aws/s3Bucket.go

```diff
@@ -46,23 +47,27 @@ func listObjects(ctx context.Context, client *s3Client.S3, bucket string, prefix
 	)
 
 	for i, p := range resp.CommonPrefixes {
-		prefix := awsSDK.StringValue(p.Prefix)
-		// Skip the top-level '/' prefix, it would be redundant to list it.
```

Comment:
Is it not really redundant? Is there going to be a directory named `#` that includes the same stuff as we list at top-level?

Reply:
Figured if the AWS console supports it, we might as well support it too if it's easy to do so. Also, you could (somehow) upload a `/` folder (e.g. something like `/<foo>/<bar>/<baz>`).

Reply:
I'm not quite sure what all's happening with CommonPrefixes and Contents. Maybe an example comment would help.